Codigo/CustomScenarios.bas

```diff
@@ -257,16 +294,19 @@ UserCanDropItem_Err:
     Call TraceError(Err.Number, Err.Description, "CustomScenarios.UserCanDropItem", Erl)
 End Function
 
-Public Sub PrepareNewEvent(ByVal eventType As e_EventType)
+Public Sub PrepareNewEvent(ByVal eventType As e_EventType, ByVal LobbyIndex As Integer)
 On Error GoTo PrepareNewEvent_Err:
```

Comment:
We should add `debug.assert LobbyIndex < UBOUND(LobbyList)`

Reply:
done